demisto_sdk/commands/common/hook_validations/test_playbook.py

```diff
@@ -1,12 +1,22 @@
 from demisto_sdk.commands.common.hook_validations.content_entity_validator import \
     ContentEntityValidator
+from demisto_sdk.commands.common.errors import Errors
+from demisto_sdk.commands.common.tools import is_string_uuid
 
 
 class TestPlaybookValidator(ContentEntityValidator):
     """TestPlaybookValidator is designed to validate the correctness of the file structure we enter to content repo for
     both test playbooks and scripts.
     """
 
+    def is_valid_test_playbook(self, validate_rn: bool = True):
```

Comment:
TPB default val of rn should be false, as customers do not care when they are changed.
```suggestion
    def is_valid_test_playbook(self, validate_rn: bool = False):
```

Reply:
OK you are right. also when this function is called the given value of this argument is false

Reply:
Indeed. but to not make this func prone to errors, best to give default vals of the most common usage, which in this case is `False`.